example/lib/config.dart.example

```diff
@@ -1,18 +1,29 @@
 // Your Situm user and API Key.
 // From 3.1.0 version onwards, situmUser is not needed anymore.
 // const situmUser = "YOUR-SITUM-USER";
-const situmApiKey = "YOUR-SITUM-API-KEY";
+const SITUM_API_KEY = "YOUR-SITUM-API-KEY";
 
 // Set here the building identifier you want on the map.
-const buildingIdentifier = "YOUR-SITUM-BUILDING-IDENTIFIER";
+const BUILDING_IDENTIFIER = "YOUR-SITUM-BUILDING-IDENTIFIER";
+
 // Alternatively, you can set an identifier that allows you to remotely configure all map settings.
 // For now, you need to contact Situm to obtain yours.
-const remoteIdentifier = null;
+const REMOTE_IDENTIFIER = null;
+
+//You can also specify the language in which you want the UX of the viewer to be displayed.
+const LANGUAGE = null;
```

Comment:
Aqui penso que podiamos deixar a constante cun valor por defecto correcto, e dicir, `const LANGUAGE = 'en';`.
Penso que queda mais coherente e entendible asi.

Reply:
Non estou seguro si un valor por defecto aporta algo. Creo que con null iría ao idioma do dispositivo, que creo que é o máis adecuado.

O que se pode é poñer algún enlace para ver cales son os idiomas soportados.

Reply:
Nese ficheiro de documentacióno final hay un enlace a doc de como obter estos parámetros. Eu creo que o mellor he deixalo a null e que colla o dioma do teléfono